crates/bevy_pbr/src/light.rs

```diff
@@ -125,7 +180,7 @@ impl Default for SpotLight {
         // a quarter arc attenuating from the center
         Self {
             color: Color::rgb(1.0, 1.0, 1.0),
-            intensity: 800.0, // Roughly a 60W non-halogen incandescent bulb
+            intensity: 4000.0, // Roughly a 300W non-halogen incandescent bulb
```

Comment:
could we use a led light instead of an incandescent bulb in the comment? 
4000 lumens seems a bit high, not your off the shelf lightbulb